src/modules/navigation/modals/CreateTable.vue

```diff
@@ -178,14 +178,19 @@ export default {
 		async loadTemplatesFromBE() {
 			try {
 				const res = await axios.get(generateUrl('/apps/tables/table/templates'))
-				if (res.status !== 200) {
-					showWarning(t('tables', 'Sorry, something went wrong.'))
-					console.debug('axios error', res)
-				}
 				this.templates = res.data
 			} catch (e) {
+				const res = e.response
+				if (res.status === 401) {
+					showError(t('tables', 'Could not load templates, not authorized. Are you logged in?'))
+				} else if (res.status === 403) {
+					showError(t('tables', 'Could not load templates, no permissions.'))
+				} else if (res.status === 404) {
+					showError(t('tables', 'Could not load templates, resource not found.'))
+				} else {
+					showError(t('tables', 'Could not load templates, unknown error.'))
+				}
 				console.error(e)
```

Comment:
This seems to be repeated quite a bit. How about introducing a helper function to handle axios errors?
```suggestion
				handleAxiosError(e, t('tables', 'Could not load templates'))
```

The helper would look somewhat like this:

```javascript
function handleAxiosError(e, message) {
	const status = e.response?.status
        showError(message + ', ' + statusMessage(status))
}

function statusMessage(status)
	if (status === 401) {
		return t('tables', 'not authorized. Are you logged in?')
	} else if (status === 403) {
		return t('tables', 'no permissions.')
	} else if (res.status === 404) {
		return t('tables', 'resource not found.')
	} else {
		return t('tables', 'unknown error.')
	}
	console.error(e)
}

Reply:
https://github.com/nextcloud/collectives/blob/0c7f87167cf460c0bdaff46a152859f0e17bd643/src/util/displayError.js#L27 might server as an example.